### PR TITLE
Feat multi select dialog

### PR DESCRIPTION
**Pull request by blanchco**
# Description

* Reusable Multi Select component
* used in project overview table for now - it can be used in other table in the future if need be
* ability to open multi tabs for now, more actions can be added in the future

https://github.com/DARPA-ASKEM/Terarium/assets/33158416/7bc4d217-baa0-48a0-b95d-9d5aa4aea971

### CODE REVIEW

**Comment by mwdchang** [comment]
@blanchco Please note I am fixing some of the tab opening/closing logic, as you are changing the tab-store we should sync up. See my PR here https://github.com/DARPA-ASKEM/Terarium/pull/1592

**Comment by blanchco** [comment]
> @blanchco Please note I am fixing some of the tab opening/closing logic, as you are changing the tab-store we should sync up. See my PR here #1592

yes let's sync up - I could also wait to merge this until you PR is in to make the changes on my side

File: packages/client/hmi-client/src/components/widgets/tera-multi-select-modal.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	</Dialog>
+</template>
+
+<script setup lang="ts">
+import Button from 'primevue/button';
+import Dialog from 'primevue/dialog';
+
+defineProps<{
+	isVisible: boolean;
+	selectedResources: any[];
```
Comment (tangopapatime) [inline packages/client/hmi-client/src/components/widgets/tera-multi-select-modal.vue:31]: we might want to try and type this properly

**Review by tangopapatime — APPROVED** [review]
LGTM

**Pull request merged**